### PR TITLE
Kubernetes playbooks for mediawiki

### PR DESCRIPTION
**Pull request by rthallisey**

### CODE REVIEW

**Comment by rthallisey** [comment]
[(13) Agnostic Runtime phase 3](https://trello.com/c/7grfQIbZ/439-13-agnostic-runtime-phase-3)

File: playbooks/provision.yml [hunk 1]
```diff
@@ -5,4 +5,7 @@
   roles:
   - role: ansible.kubernetes-modules
     install_python_requirements: no
-  - provision-mediawiki123-apb
+  - mediawiki
+  vars:
+    action: provision
+    cluster: openshift
```
Comment (djzager) [inline playbooks/provision.yml]: In the other PR you used `kubernetes` as the default, why the difference?

File: roles/mediawiki/tasks/update.yml [hunk 2]
```diff
@@ -48,8 +53,8 @@
       - name: MEDIAWIKI_ADMIN_PASS
         value: "{{ mediawiki_admin_pass }}"
       - name: MEDIAWIKI_SITE_SERVER
-        value: '{{ route.route.spec.host }}'
-      image: docker.io/ansibleplaybookbundle/mediawiki123:latest
+        value: '{{ endpoint.endpoints.subsets[0].addresses[0].ip }}'
+      image: docker.io/jmontleon/mediawiki123:update
```
Comment (djzager) [inline roles/mediawiki/tasks/update.yml]: This doesn't look like a change you want to make.

Comment (djzager) [reply roles/mediawiki/tasks/update.yml]: Shouldn't we be using the `mediawiki` image now instead of `mediawiki123`?

Comment (rthallisey) [reply roles/mediawiki/tasks/update.yml]: My rebase scooped this up. It should be docker.io/ansibleplaybookbundle/mediawiki123:latest

File: playbooks/provision.yml (hunk 1 above)
Comment (rthallisey) [reply playbooks/provision.yml]: it should be kubernetes

File: roles/mediawiki/tasks/update.yml (hunk 2 above)
Comment (djzager) [reply roles/mediawiki/tasks/update.yml]: What I meant was the [`mediawiki123` image](https://hub.docker.com/r/ansibleplaybookbundle/mediawiki123/) isn't being updated anymore, you should probably use the [`mediawiki` image](https://hub.docker.com/r/ansibleplaybookbundle/mediawiki-apb/)

**Pull request merged**